lib/rules/pseudo-element.js

```diff
@@ -12,20 +12,38 @@ var pseudoElements = yaml.safeLoad(
       fs.readFileSync(path.join(__dirname, '../../data', 'pseudoClasses.yml'), 'utf8')
     ).split(' ');
 
+/**
+ * Get prefix free version of string
```

Comment:
This isn't a great comment, Maybe 'Checks if the supplied argument is a string and contains a prefix, if it does we remove the prefix'.